modules/era/lua/era_status_effects.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- Common Requires
```

Comment:
No need for all the extra headers, this module is a good example:
```lua
-----------------------------------
-- Pre-RMT Countermeasure Drops
-----------------------------------
require('modules/module_utils')
require('scripts/globals/treasure')
-----------------------------------
local m = Module:new('pre_rmt_drops')

m:addOverride('xi.zones.Castle_Oztroja.Zone.onInitialize', function(zone)
    xi.treasure.treasureInfo[xi.treasure.type.COFFER].zone[xi.zone.CASTLE_OZTROJA].item = { 0.150, xi.item.ASTRAL_RING }
end)

return m
```

and then information about the date of the changes could go in the title block at the top